### PR TITLE
STSecurity - Carthage Support

### PR DESCRIPTION
**Pull request by ConEliopoulos**
Updated for Carthage support

@TabDigital/mobile

### CODE REVIEW

File: STSecurity.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -372,6 +456,161 @@
...
+		};
+		BFF97AA91B2AB16300A11FDA /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++0x";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
```
Comment (cysp) [inline STSecurity.xcodeproj/project.pbxproj]: Can some of these go away, are they defined in the `.xcconfig`s?

Comment (ConEliopoulos) [inline STSecurity.xcodeproj/project.pbxproj]: Hmmm... They are an eyesore

I will look into it

**Comment by cysp** [comment]
The source repo for this already builds a `.framework`.

**Comment by ConEliopoulos** [comment]
@cysp Oh. Well, lucky I didn't spend too much time on this one!

I cloned the source repo and it has one of the problems mine had. If you build the static lib for iOS device it fails with error `CodeSign error: code signing is required for product type 'Unit Test Bundle' in SDK 'iOS 8.3'`

I fixed that is my branch by turning off tests in the build phase for the STSecurity (lib) scheme.

So I guess we can just delete this branch and update master 😀

**Comment by ConEliopoulos** [comment]
Original source already has a framework, so this branch is not required. PR closed.

**Pull request closed without merging**